packages/hooks/src/useMaxWithdrawAmount.ts

```diff
@@ -0,0 +1,8 @@
+import { useStore } from '@deriv/stores';
+
+const useMaxWithdrawAmount = () => {
+    const { client } = useStore();
+    return client.getLimits()?.get_limits?.remainder;
```

Comment:
```suggestion
    const { client } = useStore();
    
    return client.getLimits()?.get_limits?.remainder;
```
@hamid-deriv Can we make it non-nullable? Like returning `0` for default? 🤔

Reply:
Fixed.